docs/deprecations.md

```diff
@@ -6,6 +6,13 @@ layout: Doc
 
 # Serverless Framework Deprecations
 
+## Http Api provider tags
+
+Deprecation code: `AWS_HTTP_API_USE_PROVIDER_TAGS`
+
+Starting with v3.0.0, provider tags will be applied to Http Api Gateway by default
```

Comment:
Let's use `provider.tags` instead of "provider tags" phrase here to highlight that it's a part of configuration. It should be: "Starting with v3.0.0, `provider.tags` will be applied to Http Api Gateway by default"

Reply:
just pushed, thanks for the detailed comment and sorry for missing that